client/dist/style.css

```diff
@@ -0,0 +1,168 @@
+body{
+  background-color: #041c2c;
+  font-family: SystemFont,Helvetica Neue,Helvetica,Arial,sans-serif;
+  line-height: 1.3;
+}
+
+.reviewlist_container{
+  width: 100%;
+  box-sizing: border-box;
+  display:flex;
+  height: 240px;
+  }
+
+
+
+.review_entry {
+  width: 300px;
+  margin-right: 20px;
+  height: 240px;
+  box-sizing: border-box;
+  color: #fff;
+  font-family: sans-serif;
+  font-size: 17px;
+  display: inline-block;
+  overflow: hidden;
+  white-space: normal;
+
+}
+
+.review_text {
+    font-size: 15px;
+    text-align: left;
+    padding-left: 27px;
+    padding-bottom: 30px;
+    box-sizing: border-box;
+   }
+
+.author {
+    font-size: 17px;
+}
+.source {
+  font-size: 12px;
+  text-align: left;
+  padding-left: 27px;
+  color: #b0bec5;
+  vertical-align: top;
+  font-style: italic;
+}
+ .rating_high{
+   color: red;
+  }
+ .rating_low{
+  color: #00FF00;
+}
+
+#review_carousel{
+  width: fit-content;
+  height: 480px;
+  transition: transform .5s ease;
+  white-space: nowrap;
+}
+
+#p_backward, #m_backward{
+  background-image: url(https://www.vudu.com/content/images/pov_left_arrow.svg);
+  background-position: center;
+  background-repeat: no-repeat;
+  position: absolute;
+  z-index: 100;
+  height: 480px;
+  width: 35px;
+  background-color: rgba(0, 0, 0, 0.582);
+  display: none;
+}
+
+
+#p_forward, #m_forward{
+  background-image: url(https://www.vudu.com/content/images/pov_right_arrow.svg);
+  background-position: center;
+  background-repeat: no-repeat;
+  position: absolute;
+  z-index: 99;
+  height: 480px;
+  width: 35px;
+  background-color: rgba(0, 0, 0, 0.582);
+  display: none;
+  top: 60px;
+  right: 0px;
+}
+
+
+.review_container:hover #p_backward{

```

Comment:
You can combine these four `:hover` pseudo-classes when using styled-components.
EDIT: Saw that you did so later on, good stuff!